app/services/cache/ExportsCacheModelRepository.scala

```diff
@@ -77,7 +78,11 @@ case class ExportsCacheModel(
   dispatchLocation: Option[DispatchLocation] = None,
   additionalDeclarationType: Option[AdditionalDeclarationType] = None,
   consignmentReferences: Option[ConsignmentReferences] = None,
-  exporterDetails: Option[ExporterDetails] = None,
+  warehouseIdentification: Option[WarehouseIdentification] = None,
+  borderTransport: Option[BorderTransport] = None,
+  transportDetails: Option[TransportDetails] = None,
+  containerData: Option[TransportInformationContainerData] = None,
+  parties: Option[Parties] = None,
```

Comment:
It looks to me like `Parties` is just  a container class for some Optional fields. 
Why don't we change this to `parties: Parties = Parties()`

Reply:
Super Cool!!! :) Thanks 
Addressed!